Library/Formula/pmd.rb

```diff
@@ -1,8 +1,20 @@
 class Pmd < Formula
   desc "Source code analyzer for Java, JavaScript, and more"
   homepage "http://pmd.sourceforge.net/"
-  url "https://github.com/pmd/pmd/releases/download/pmd_releases/5.3.3/pmd-src-5.3.3.zip"
-  sha256 "38004ea3274d2f71701f438606a4c4095b7a86110e8d7f2a8940170c9bd5ddbb"
+  head "https://github.com/pmd/pmd.git"
+
+  stable do
+    url "https://github.com/pmd/pmd/releases/download/pmd_releases/5.4.1/pmd-src-5.4.1.zip"
+    sha256 "1dcef1a9f6e7681e474faf2d37ed82b408edf880b3707f17f42b0046cf3130a1"
+
+    # Some versions of PMD have incorrect versions specified for parent POM's.
```

Comment:
The maintainer is aware of the general issue - there's even a section in his release script reminding him to fix it, and there are bug reports for past versions on Sourceforge. I just concluded that this has a good chance of being a recurring issue, so we might as well get used to patching now.

On January 11, 2016 1:41:41 AM PST, Dominyk Tiller <notifications@github.com> wrote:
>> @@ -1,8 +1,20 @@
>>  class Pmd < Formula
>>    desc "Source code analyzer for Java, JavaScript, and more"
>>    homepage "http://pmd.sourceforge.net/"
>> -  url
>"https://github.com/pmd/pmd/releases/download/pmd_releases/5.3.3/pmd-src-5.3.3.zip"
>> -  sha256
>"38004ea3274d2f71701f438606a4c4095b7a86110e8d7f2a8940170c9bd5ddbb"
>> +  head "https://github.com/pmd/pmd.git"
>> +
>> +  stable do
>> +    url
>"https://github.com/pmd/pmd/releases/download/pmd_releases/5.4.1/pmd-src-5.4.1.zip"
>> +    sha256
>"1dcef1a9f6e7681e474faf2d37ed82b408edf880b3707f17f42b0046cf3130a1"
>> +
>> +    # Some versions of PMD have incorrect versions specified for
>parent POM's.
>
>Is this something worth reporting upstream at all, or?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/Homebrew/homebrew/pull/47902/files#r49302411

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.